src/ui/handler/scroll_zoom.js

```diff
@@ -129,16 +133,20 @@ class ScrollZoomHandler {
      *
      * @param {Object} [options] Options object.
      * @param {string} [options.around] If "center" is passed, map will zoom around center of map.
+     * @param {boolean} [options.requireCtrl] If set to true, ctrl key must be pressed while scrolling to zoom.
      *
      * @example
      * map.scrollZoom.enable();
      * @example
      * map.scrollZoom.enable({around: 'center'});
+     * map.scrollZoom.enable({requireCtrl: true});
```

Comment:
Does this need another @example tag?

```suggestion
     * @example
     * map.scrollZoom.enable({requireCtrl: true});
```

Reply:
nice catch